src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -115,9 +115,6 @@ def _configure_db_create_params(
     create_mode: Valid CreateMode enum value (e.g. `default`, `copy`, etc)
     """
 
-    # This line to be removed when zone redundancy is fully supported in production.
-    arg_ctx.ignore('zone_redundant')
```

Comment:
is zone_redundant allowed for all create modes (copy, geo secondary, restore)? is it allowed for DW?

Reply:
Nope, so I had to block the different create modes and dw